.github/workflows/website.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          repository: 'tridactyl/site'
+            # submodules: 'recursive' # we shouldn't need this, public should be generated each time
+          path: 'site'
+      - name: Build
+        run: |
+          cd tridactyl
+          yarn install
+          yarn run build
+          find . -iname "*.html" -exec sed 's@href="/static@href="/tridactyl/public/build/static@' -i '{}' ';' # ideally this url would be less gnarly
+          cd ../site
+          sudo apt-get install -y hugo
+          hugo
+          cp -r ../tridactyl/build/ public
```

Comment:
Are you meaning to include all of Tridactyl here? I think all the site stuff is just in static/

We could get some kind of cut down version of Tridactyl running, but we would need to change the scripts tags to in head to point to `/[x]/public/build/static/docs/modules/tridactyl-docs/public/build/content.js` and also build Tridactyl with a dummy `browser` object somehow, which seems like it could be cool but also a lot of work?

Reply:
>We could get some kind of cut down version of Tridactyl running

This was precisely what I was thinking about :) Indeed it'd probably be a lot of work and if I recall correctly is not required to get simple HTML docs working.